flake.lock

```diff
@@ -1,5 +1,17 @@
 {
   "nodes": {
+    "flake-registry": {
+      "flake": false,
+      "locked": {
+        "narHash": "sha256-Wz8gOVJbyCBrRXuBot9IEsdzIPnu2ZRl3uH9+D3/uIc=",
+        "type": "file",
+        "url": "https://channels.nixos.org/flake-registry.json"
```

Comment:
We can't lock this file because it's mutable and not a redirect. Probably better to use the underlying git repo (https://github.com/NixOS/flake-registry).

Reply:
@edolstra Do my latest changes resolve this concern?